test/e2e/annotations/disableproxyintercepterrors.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Ditto

Reply:
Well .. can't argue here. Just copied that header from a file that old.

Reply:
Fixed